FizzBuzz - single parameter.rb

```diff
@@ -0,0 +1,47 @@
+def fizzBuzz(isFB)
```

Comment:
while having a general method to call to start program is a nice idea, I think you went a little bit too far here. You don't need to define methods in methods. Just the `if isNumber(isFB)` part would serve the purpose.

Reply:
Just to be sure I am understanding this suggestion correctly: I should unwrap my code from the `fizzBuzz` method and toss it altogether? Or are you referring only to the `isNumber` - suggesting I squeezed it's task into the following `if isNumber(isFB)`?

Reply:
What Ibwas trying to say is that you can define the methods like `is_number` _outside_ of the `fizz_buzz` method. This way `fizz_buzz` can use them, while still containing only the conditional for doing the actual work :)

So we're looking for something like this:

```ruby
def is_number(string)
  # code
end

def fizz_buzz
  if is_number
    # code
  else
    # error handling
  end
end
```

Instead of:

```ruby 

def fizz_buzz(input)
  def is_number(string)
    # code
  end

  if is_number
    # code
  else
    # error handling
  end
end
```